ansible/templates/hosts

```diff
@@ -0,0 +1,13 @@
+; the first parameter in a host is the inventory_hostname which has to be
```

Comment:
fixed, template has been moved to ansible/environments/distributed/templates/hosts.j2